#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       name    type     level     description     default      min      max
gen.add("text_threshold", double_t, 0, "Text confidence threshold", 0.70, 0.0, 1.0)
gen.add("text_low_bound_score", double_t, 0, "Text low bound score", 0.40, 0.0, 1.0)
gen.add("link_threshold", double_t, 0, "Link confidence threshold", 0.1, 0.0, 1.0)
gen.add("mag_ratio", double_t, 0, "Image magnification ratio", 1.5, 0.1, 10.0)
gen.add("max_image_size", int_t, 0, "Max image size for inference", 1280, 128, 4096)

exit(gen.generate(PACKAGE, PACKAGE, "CRAFT"))
